plugin/src/main/java/org/screamingsandals/bedwars/Main.java

```diff
@@ -553,7 +553,7 @@ public void enable() {
         /* Initialize our ScoreboardLib*/
         ScoreboardManager.init(this.getPluginDescription().as(JavaPlugin.class));
 
-        PlayerMapper.getConsoleSender().sendMessage(Component.text("Everything is loaded! If you like our work, consider visiting our Patreon! <3").color(NamedTextColor.WHITE));
+        PlayerMapper.getConsoleSender().sendMessage(Component.text("everything has finished loading! If you like our work, consider subscribing to our Patreon! <3").color(NamedTextColor.WHITE));
```

Comment:
It's start of a sentence so big E please xdd. And consider checking the base English language file, this file is not updated via Crowdin (don't update keys, only values tho)

Reply:
EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE